bindings/nodejs/generated.d.ts

```diff
@@ -32,6 +32,35 @@ export interface ListOptions {
   limit?: number
   recursive?: boolean
 }
+export interface OpWriteOptions {
```

Comment:
Thanks for you effort. But `OpXxx` is not an existing concept at the binding side. How about using `WriteOptions`?

Reply:
IMHO, the `OpWriteOptions` concept corresponds to `OpWrite`, so I think it's better to use one object not `OpWriteOptions`, `OpWriterOptions`

the name starts with `Op` also because of this, but `WriteOptions` is ok for me

Reply:
> IMHO, the `OpWriteOptions` concept corresponds to `OpWrite`, so I think it's better to use one object not `OpWriteOptions`, `OpWriterOptions`

Your understanding is correct. But there is a gap between rust core and nodejs binding.

Rust Core users have a concept called `OpXxx` like `OpWrite` and `OpRead`. But we never expose those APIs to nodejs binding. So from the nodejs users view, they don't know about `OpXxx` thus they can't understand what's `OpXxx` represents for.

So I think it's better to use `WriteOptions` without the `Op` prefix.

And for `WriteOptions` and `WriterOptions`, it's possible for the future to introduce different options inside `write` and `writer` options. So I want to seperate them for now instead of another breaking changes in the future.

Reply:
Thanks, that's a comprehensive explanation, I will fix it ASAP